Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM frolvlad/alpine-python3
```

Comment:
>  guess the reason why it still used Python 2 is because the python-pip package depends on Python 2.

That seemed to be what was happening, yes. I'd apk install Python 3 and _not_ Python 2, but then my `setup.py` would get run in Python 2 and crash, presumably because it was Python 2 pip that got installed.

Looks like nick has the solution, I'll open an issue so that I remember to fix this